tasmota/xsns_14_sht3x.ino

```diff
@@ -79,6 +80,11 @@ bool Sht3xRead(float &t, float &h, uint8_t sht3x_address)
 
 void Sht3xDetect(void)
 {
+#ifdef USE_M5EPD47
+  sht3x_wire = &Wire1;
+#else
+  sht3x_wire = &Wire;
+#endif
   for (uint32_t i = 0; i < SHT3X_MAX_SENSORS; i++) {
```

Comment:
This construction is a workaround for the second wire bus only available on esp32.

Your implementation is not fool proof; all calls too ``I2cxxxx()`` are still using bus0 only.

We need to find a better way to support two I2C busses.

Reply:
of course i know it is only a quick workaround and in this case it works because the detect device mechanism is different than in other drivers.
here is a proposal how to do it probably better.  i implemented this for mlx90614

[xsns_46_MLX90614.ino.zip](https://github.com/arendst/Tasmota/files/7449154/xsns_46_MLX90614.ino.zip)